scipy-1.0/paper.tex

```diff
@@ -764,23 +772,15 @@ \subsection*{Language choices}
     \end{axis}
     \end{tikzpicture}
     \caption{The breakdown of programming languages used in the
-             SciPy library determined using the linguist library.
+             SciPy library determined using the \texttt{linguist} library.
     	 Small ($<0.5 \%$) amounts of TeX, Matlab, Shell,
     	 and Makefile are excluded for clarity and mostly
     	 provide supporting roles in tests, building, and
     	 documentation.}
     \label{fig:linguist}
 \end{figure}
 
-For implementing new functionality, we have a clear order of language
-preference.  First Python, if performance is not an issue. If it is, then in
-order of preference: Cython, C, C++, Fortran. The main motivation for this is
-maintainability: Cython has the highest abstraction level and most Python
-developers will understand it. C is also widely known, and easier to deal with
-for the core development team than C++ and especially Fortran.
-
-While it is not surprising that Python is heavily used in SciPy, the usage
-distribution of the other languages warrants some discussion. Fortran is an
+Fortran is an
 extremely well-established scientific programming language, both for historical
 reasons and because of its continued excellent
 performance\cite{Koelbel:1993:HPF:562354}. We wrap the FFTPACK Fortran library
```

Comment:
Do we need to establish the excellence of these well-known packages? I would just go about it with 

```
Fortran, despite it is age, is still a high-performance scientific programming language with continued contemporary usage. Thus, we wrap the following excellent field-tested Fortran libraries in order to provide the Python convenience while still benefiting from their respective performance: THISPACK \cite{...}, THATPACK \cite{...} ...
```

We can still give some praising comments but these are kind-of known assets out there.

Reply:
makes sense to me